addons/mrp_account/views/mrp_workcenter_views.xml

```diff
@@ -0,0 +1,12 @@
+<odoo>
+    <record id="mrp_workcenter_tree_view_inherited" model="ir.ui.view">
+        <field name="name">mrp.workcenter.tree.inherited</field>
+        <field name="model">mrp.workcenter</field>
+        <field name="inherit_id" ref="mrp.mrp_workcenter_tree_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='costs_hour']" position="after">
+                <field name="costs_hour_account_id" optional="hide"/>
```

Comment:
For the form view, this fields is only added in enterprise (mrp_account_enterprise). Maybe change move from enterprise to community for the form too ?

Reply:
I moved this part in enterprise.